js/packages/teams-ai/src/MessageExtensions.ts

```diff
@@ -597,3 +751,16 @@ function matchesPreviewAction(activity: Activity, botMessagePreviewAction?: 'edi
         return botMessagePreviewAction == undefined;
     }
 }
+
+// TODO: Refactor Test utils into testing folder (after breaking changes are completed)
```

Comment:
This file isn't being affected with Steve's breaking changes, is there a reason we want to wait for it to complete?

Reply:
Moving tests into a test folder will conflict with Steve's changes. Haven't quite thought of how I would do this export yet, but I figure once folder structure is settled a little bit I'll figure something decent out. :)